conf/standalone.conf

```diff
@@ -927,8 +927,10 @@ allowAutoSubscriptionCreation=true
 # The number of partitioned topics that is allowed to be automatically created if allowAutoTopicCreationType is partitioned.
 defaultNumPartitions=1
 
+transactionCoordinatorEnabled=true
```

Comment:
We should disable the TC by default since the transaction is not a production-ready feature for now. Some users use the standalone on their production environment which they can tolerate node failures.

Reply:
sorry, I didn't mean to commit this file. removed